fileManagment/ViewController.swift

```diff
@@ -0,0 +1,184 @@
+   
+        
+                if let name = filesTextField.text {
+            do {
+                
+                switch typeIndex {
+                    
+                case 0 :
+       
+                    if let selectedFolderName = selectedFolderName {
+
+                        
+                        let dirForFile = appFilesDirectory?.appendingPathComponent(selectedFolderName).appendingPathComponent("\(name).txt")
+          
+                              let text = contentTextField.text!.data(using: .utf8)
+          
+                              fileManager.createFile(atPath: dirForFile!.path, contents: text, attributes: nil)
+                        self.selectedFolderName = nil
+                        showAlert(message: "تم انشاء ملف جديد بداخل المجلد المختار ", title: "تمت العملية بنجاح", actionTitle: "حسنا")
+
+                        
+                    }else{
+                    
+                        showAlert(message: "الرجاء اختيار مجلد ليتم انشاء ملف بداخلة", title: "خطأ", actionTitle: "حسنا")
+
+                        
+                    }
+            
+                    
+                case 1:
+                    print("مجلد")
+                    let desiredFolderDirectory = appFilesDirectory?.appendingPathComponent(name)
+
+                    try   fileManager.createDirectory(at: desiredFolderDirectory!, withIntermediateDirectories: true, attributes: [:])
+                    
+                    showAlert(message: "تم انشاء مجلد جديد", title: "تمت العملية بنجاح", actionTitle: "حسنا")
+
+                    
+                default:
+                    let desiredFolderDirectory = appFilesDirectory?.appendingPathComponent(name)
+
+                    try   fileManager.createDirectory(at: desiredFolderDirectory!, withIntermediateDirectories: true, attributes: [:])
+
+                    let dirForFile = appFilesDirectory?.appendingPathComponent(name).appendingPathComponent("Note.txt")
+
+                    let text = contentTextField.text!.data(using: .utf8)
+
+                    fileManager.createFile(atPath: dirForFile!.path, contents: text, attributes: nil)
+                    
+                  showAlert(message: "تم انشاء مجلد و ملف جديد", title: "تمت العملية بنجاح", actionTitle: "حسنا")
+                    
+                }
+                
+                
+
+            } catch  {
+
+            }
+            relodFiles()
+            fileTableView.reloadData()
+
+        }else{
+
+        }
+        
+    }
+    
+    @IBAction func segmentUpdate(_ sender: Any) {
+        
+        if actionTypeSegment.selectedSegmentIndex == 1 {
+     
+            contentTextField.isHidden = true
+
+
+        }else{
+            contentTextField.isHidden = false
+        }
+        
+    }
+    
+    
+    func showAlert(message : String , title : String, actionTitle: String)  {
+        
+        let alert = UIAlertController(title: title, message: message, preferredStyle: .alert)
```

Comment:
you have used the alert in such a nice way! as well as making it reusable, great job!